output.proto

```diff
@@ -74,6 +74,7 @@ message StreamMessage {
   repeated Object objects = 1;
   repeated ZoneConfig zones = 101;
   SystemHealth health = 102;
+  google.protobuf.Timestamp cloud_arrival_timestamp = 103;
```

Comment:
I believe cloud_arrival_timestamp will be set every frame. Then please assign index to `2`.
And if possible, Could you add a comment like `//oldest point cloud timestamp in all algo nodes`
```suggestion
  google.protobuf.Timestamp cloud_arrival_timestamp = 2;
```

Reply:
Thanks for the suggestion. I was unsure of the ID. How does `first_cloud_arrival_timestamp` sound? I'd like to minimize comments if there's a better variable name.

Reply:
Yes, it sounds OK for me.
Maybe we better to listen the other people opinion in your argos PR.